d3/d3.d.ts

```diff
@@ -962,7 +966,7 @@ declare module D3 {
                 */
                 (elements: EventTarget[]): Transition;
             }
-            each: (type?: string, eachFunction?: (data: any, index: number) => any) => Transition;
+            each: (type?: any, eachFunction?: (data: any, index: number) => any) => Transition;
```

Comment:
I think `type?: string` is collect.
https://github.com/mbostock/d3/blob/8a0fe7f2d206c817c1ecb61b64dc6d6ac3573fbe/src/transition/each.js#L22
Maybe we need a overload definition.